CapstoneOnGoing/Controllers/OnGoingController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using CapstoneOnGoing.Logger;
+using CapstoneOnGoing.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Models.Request;
+
+namespace CapstoneOnGoing.Controllers
+{
+	[Route("api/ongoing")]
+	[ApiController]
+	public class OnGoingController : ControllerBase
+	{
+		private readonly IMapper _mapper;
+		private readonly IUserService _userService;
+		private readonly ILoggerManager _logger;
+
+		public OnGoingController(IMapper mapper, IUserService userService, ILoggerManager logger)
+		{
+				_mapper = mapper;
+				_userService = userService;
+				_logger = logger;
+		}
+
+		//[Authorize(Roles = "ADMIN")]
+		[HttpPost]
+		public IActionResult ImportInProgressStudents([FromBody]IEnumerable<InProgressStudentsRequest> inProgressStudentsRequest)
+		{
+			bool isImportSuccessfully = _userService.ImportInProgressStudents(inProgressStudentsRequest);
+			if (isImportSuccessfully)
+			{
+				return Ok("Import in-progress students successfully");
```

Comment:
(Optional) Nên gửi về json dạng:
{
    "status": 200,
    "message": "Import in-progress students successfully",
    "timestamp": 37413286154
}